src/main/java/onboarding/Problem1.java

```diff
@@ -3,8 +3,71 @@
 import java.util.List;
 
 class Problem1 {
+
+    public static int sum(int x){
+        int result_sum = 0;
+        int result_mul = 1;
+        while (x!=0){
+            result_sum+=(x%10);
+            result_mul*=(x%10);
+            x/=10;
+        }
+        return Math.max(result_sum,result_mul);
+    }
+
+    public static class PageOrderException extends Exception{
+        public PageOrderException(){ }
+        public PageOrderException(String message){
+            super(message);
+        }
+    }
+
+    public static class Gamer{
+        int left,right,max_num;
```

Comment:
이렇게 인라인 스타일로 정의한 필드 보다는
```suggestion
        int left;
        int right;
        int maxNum;
```